nbconvert/preprocessors/tests/test_execute.py

```diff
@@ -195,19 +197,36 @@ def timeout_func(source):
 
     def test_allow_errors(self):
         """
-        Check that conversion continues if ``allow_errors`` is False.
+        Check that conversion halts if ``allow_errors`` is False.
         """
         current_dir = os.path.dirname(__file__)
         filename = os.path.join(current_dir, 'files', 'Skip Exceptions.ipynb')
         res = self.build_resources()
         res['metadata']['path'] = os.path.dirname(filename)
         with pytest.raises(CellExecutionError) as exc:
             self.run_notebook(filename, dict(allow_errors=False), res)
-        self.assertIsInstance(str(exc.value), str)
-        if sys.version_info >= (3, 0):
-            assert u"# üñîçø∂é" in str(exc.value)
-        else:
-            assert u"# üñîçø∂é".encode('utf8', 'replace') in str(exc.value)
+            self.assertIsInstance(str(exc.value), str)
+            if sys.version_info >= (3, 0):
+                assert u"# üñîçø∂é" in str(exc.value)
+            else:
+                assert u"# üñîçø∂é".encode('utf8', 'replace') in str(exc.value)
+
+    def test_raises_exception_cell_tag(self):
+        """
+        Check that conversion continues if ``cell_tags`` is False.
```

Comment:
This docstring doesn't seem quite right.

Reply:
agreed.